### ISSUE TITLE
Inquiry of robustness of model with humanoid robot

### ISSUE THREAD

**Issue by PhilipAmadasun** [google-deepmind/open_x_embodiment#51]
Has the model been trained for use with humanoid robots with the ability to walk and hold objects?

**Comment by open-x-embodiment** [comment]
No, the RT-*-X models have not been trained with humanoids. That being
said, the models are trained with end-effector delta as the action space.
So if you have the data from humanoids, it will be very useful if you share
it with the community or try finetuning the model with your data.

**Comment by PhilipAmadasun** [comment]
@open-x-embodiment Thanks for the feedback. Would you happen to know of any models that already have some pretraining on this?

**Comment by XibalbaTechSol** [comment]
Couldn't one create synthetic data using computer models of various robots?
